RELEASE.md

```diff
@@ -222,13 +243,15 @@ target component.
     $ hab pkg exec core/hab-bintray-publish publish-studio
     ```
 
-1. Publish the Linux and Mac artifacts by selecting the appropriate `.hart` file
+1. Publish the Linux, Mac and Windows artifacts by selecting the appropriate `.hart` file
 
     ```
     $ hab pkg exec core/hab-bintray-publish publish-hab \
       ./results/core-hab-0.10.2-20160930230245-x86_64-linux.hart
     $ hab pkg exec core/hab-bintray-publish publish-hab \
       ./habitat/components/hab/mac/results/core-hab-0.10.2-20160930230245-x86_64-darwin.hart
+    $ hab pkg exec core/hab-bintray-publish publish-hab \
```

Comment:
We'll need to add a branch to the case statement of publish-hab (https://github.com/habitat-sh/habitat/blob/master/components/bintray-publish/bin/publish-hab.sh#L205).